### PR TITLE
more motivation lines!

### PR DESCRIPTION
**Pull request by jpaezg93**

### CODE REVIEW

**Comment by pepitoperez93** [comment]
great motivation lines! Now, let's try to change the first letters to capital

File: testing_file.py
Comment (pepitoperez93) [inline testing_file.py:1]: please change all the letters at the beggining of sentences to capital letters

Comment (pepitoperez93) [reply testing_file.py:1]: also as an additional practice add points to all end of sentences

**Comment by jpaezg93** [comment]
We solved all comments.

**Review by pepitoperez93 — CHANGES_REQUESTED** [review]
please add one more motivational sentence

**Review by pepitoperez93 — APPROVED** [review]

**Pull request merged**